### PR TITLE
wafw00f: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/wafw00f.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`wafw00f --findall {{https://www.example.com}}`
+
+- Pass requests through a [p]roxy (such as BurpSuite):
+
+`wafw00f --proxy {{http://localhost:8080}} {{https://www.example.com}}`
+
+- [t]est for a specific WAF product (run `wafw00f -l` to get list of all supported WAFs):
+
+`wafw00f --test {{Cloudflare|Cloudfront|Fastly|ZScaler}} {{https://www.example.com}}`
```
Comment (spageektti) [inline pages/common/wafw00f.md:20]:
```suggestion
`wafw00f --test {{Cloudflare|Cloudfront|Fastly|ZScaler|...}} {{https://www.example.com}}`
```

**Review by spageektti — APPROVED** [review]
LGTM, thanks for your contribution.

File: pages/common/wafw00f.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# wafw00f
+
+> Identify and fingerprint Web Application Firewall (WAF) products protecting a website.
+> More information: <https://github.com/EnableSecurity/wafw00f>.
+
+- Check if a website is using any WAF:
+
+`wafw00f {{https://www.example.com}}`
+
+- Test for [a]ll detectable WAFs without stopping at first match:
```
Comment (gutjuri) [inline pages/common/wafw00f.md:10]:
```suggestion
- Test for [a]ll detectable WAFs without stopping at the first match:
```

**Review by gutjuri — APPROVED** [review]
Thanks, I have one small suggestion :)

**Pull request merged**